spec/app/domain/authentication/authn-oidc/update_input_with_username_from_id_token_spec.rb

```diff
@@ -42,23 +45,35 @@ def request_body(request)
   end
 
   let(:authenticate_id_token_request) do
-    mock_authenticate_oidc_request(request_body_data: "id_token={\"id_token_username_field\": \"alice\"}")
+    mock_authenticate_oidc_request(request_body_data: "id_token={\"id_token_username_field\": \"alice\"}", request_headers: {"CONTENT_TYPE" => "text/plain"})
```

Comment:
Why does the test need this new content type? Doesn't that mean the API is broken from how it used to work?

Reply:
fixed